include/nighthawk/common/operation_callback.h

```diff
@@ -0,0 +1,6 @@
+#include <functional>
+
+namespace Nighthawk {
+
+using OperationCallback = std::function<void(bool, bool)>;
```

Comment:
Could you document what these two bools are?  You can add descriptive variable names for example e.g.

 std::function<void(bool done, bool ok)>;

Reply:
Done in https://github.com/envoyproxy/nighthawk/pull/130/commits/8abb45eebb97f58d202b0963b3b683ce8161c076